### ISSUE TITLE
Messages sent before session is established are not going through

### ISSUE THREAD

**Issue by cstiens** [zom/Zom-Android-XMPP#220]
* Lala sent 3 short messages that TL didn’t get

* TL sent one short message which Lala got and only after that did TL start receiving Lala’s messages

* No problems after that except that when Lala tried to take a picture directly from the chat the App crashed

version 15.1.0-alpha-1

**Comment by n8fr8** [comment]
We now set messages to queued/postponed by default, so they will send once the session and encrypted is activated.
